src/main/java/ParkingLot/AbstractEmployees.java

```diff
@@ -0,0 +1,55 @@
+package ParkingLot;
```

Comment:
package name should be in lowercase

Reply:
if i do this, git accepts them like new files, how i can do this, and see changes between commits?

Reply:
it will track them as changed files, not as new ones (in this case the only thing changed will be package parkinglot;)

Reply:
ok, done. Thank you